src/agent/input-tester/src/tester.rs

```diff
@@ -78,7 +71,6 @@ pub struct Tester {
     ignore_first_chance_exceptions: bool,
     appverif_controller: Option<AppVerifierController>,
     bugs_found_dir: PathBuf,
-    coverage_map: Option<AppCoverageBlocks>,
```

Comment:
Is the plan to add a `Option<CommandgBlockCov>` here in a future PR?

Reply:
Yeah, but I can just add that right now.

Reply:
Ah right, I take it back, it makes more sense to do this in a follow-up. Before we were loading it from a directory in the ctor, but there's more to think about now.

Reply:
https://github.com/microsoft/onefuzz/issues/712